debezium-connector-binlog/src/test/java/io/debezium/connector/binlog/BinlogConnectorIT.java

```diff
@@ -2652,6 +2655,15 @@ public void shouldEmitTruncateOperation() throws Exception {
         stopConnector();
     }
 
+    private String getExpectedQuery(String statement) {
```

Comment:
I think this should be overridden in `BinlogConnectorIT` instead of condition.

Reply:
Good point.